en/components/dock-manager.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Manage Angular Page Layout | Dock Manager | Ignite UI | Infragistics
+_description: Learn how our Dock Manager can help you in customizing the page layout of your Angular application by using unpinning, resizing, moving and hiding panes.
+_keywords: manage angular page layout, Ignite UI for Angular, Infragistics
```

Comment:
```suggestion
_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Manage Angular page layout, Dock Manager
```

Reply:
These keywords are generated following our SEO process and should remain the same.

Reply:
I have extended the keywords with the ones we use across the topics and have kept the unique ones you have added

Reply:
The SEO guidelines say that we should have only a primary keyword followed by the product and company names.